### DISCUSSION TITLE
Scroll wheel brush smoother transition

### DISCUSSION THREAD

**Discussion by CustomGraphix1** [richrobber2/canvas-zoom#96] [Ideas]
Any chance of either matching the ctrl+wheel steps to the steps on the slider so we get the same ammount of steps, currently when using ctrl+wheel the smallest is 1px the next scroll wheel size it like 20px, if you use the slider there are more steps inbetween those, its just annoying to have to keep zooming out to use the slider when we have the ctrl+wheel keybind,

If we could have the option to maybe increase by 2px per scroll click it would be perfect.

As someone that loves inpainting and advises everyone to get this extension i would really love to see this feature added!

**Comment by daswer123** [comment]
Hi, in a recent update a slider has been added that allows you to change the % change of the brush by ctr+wheel
https://github.com/richrobber2/canvas-zoom/issues/95#issuecomment-1852755003

**Reply by CustomGraphix1** [reply]
yea but the problem is the ctrl+wheel like i said doesnt match the smoothness of the steps to the slider which has more steps per slide to steps per scroll click even with the size set to 1, any chance to be able to set it as low as 0.1? that might help more.

If you look at how smooth the size changes by using the slider to just ctrl+wheel you will see what i mean

**Reply by daswer123** [reply]
Yes, you can use fractional values, the most important thing is that they should be greater than 0.

**Reply by daswer123** [reply]
UPD: For some strange technical reason, I can't get the % value below 1, I have yet to figure it out.
So for the moment 1% is the lowest value.

**Reply by CustomGraphix1** [reply]
Ahh ok, well hopefully you figure it out, its the only thing that has bugged me honestly :D 

Oh and thanks for adding the option to make prompts inactive when using the inpaint, cant tell you how many times my prompts end up with rrrrrrrrrrr trying to reset the size lol
